docs/token.go

```diff
@@ -0,0 +1,45 @@
+package docs
+
+import "github.com/netlify/gotrue/api"
+
+// swagger:route POST /token token tokenPostParams
+//
+// This is an OAuth2 endpoint that currently implements the password and refresh_token grant types.
+//
+// This is an OAuth2 endpoint that currently implements the password and refresh_token grant types. Once you have an access token, you can access the methods requiring authentication by settings the `Authorization: Bearer YOUR_ACCESS_TOKEN_HERE` header.
```

Comment:
Endpoint for handling a token from an OAuth provider. Tokens can by used by setting the `Authorization: Bearer YOUR_ACCESS_TOKEN_HERE` header in a followup request.